apps/console/src/public/deployment.config.json

```diff
@@ -924,6 +924,11 @@
                 "disabledFeatures": [],
                 "enabled": true,
                 "scopes": {}
+            },
+            "fapiApplicationCreation": {
```

Comment:
Let's add this to the J2 and add the default value in the framework side as well.

J2: https://github.com/wso2/identity-apps/blob/master/apps/console/java/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2#L1457

Default JSON: https://github.com/wso2/carbon-identity-framework/blob/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json#L1260

Reply:
Added to J2 with e2ed10aaa3c1bc125cee4996a20e8ae21906bd9c

Reply:
Added to default json with https://github.com/wso2/carbon-identity-framework/pull/5171